docs/ocis/adr/0022-sharing-and-space-management-api.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "22. Sharing and Space Management API"
+---
+
+* Status: accepted
+* Deciders: @jammingben @butonic @theonering @kobergj @micbar
+* Date: 2023-08-08
+
+Technical Story: [Public issue](https://github.com/owncloud/ocis/issues/6993)
+
+## Context and Problem Statement
+
+In the early days of the rewrite of ownCloud it was an important goal to keep all important APIs compatible with ownCloud 10. Infinite Scale embraced that goal until version 1.0.0.
+
+After that first release, the focus changed.
+
+Infinite Scale started the spaces feature which brings a whole new set of APIs and concepts. We made the conscious decision to keep the sharing API as it was, live with its shortcomings and create workarounds to support spaces. We have come a long way so far. Now we need to move on. The Web Client has made the decision to drop the support of ownCloud 10 and keep version 7.0.0 alive for ownCloud 10 to keep the easy migration path intact.
```

Comment:
Nitpick, but technically Web is keeping version 7.0 alive. We are already at 7.0.3 currently.

```suggestion
Infinite Scale started the spaces feature which brings a whole new set of APIs and concepts. We made the conscious decision to keep the sharing API as it was, live with its shortcomings and create workarounds to support spaces. We have come a long way so far. Now we need to move on. The Web Client has made the decision to drop the support of ownCloud 10 and keep version 7.0 alive for ownCloud 10 to keep the easy migration path intact.
```